media/js/base/site.js

```diff
@@ -6,36 +6,11 @@
     'use strict';
     window.site = {
         getPlatform: function (ua, pf) {
-            // Firefox OS navigator.platform is an empty string, which equates to a falsey value in JS
-            // Ths means we must use an ugly ternary statement here to make testing easier.
-            pf = (pf === '') ? '' : pf || navigator.platform;
+            pf = pf || navigator.platform;
             ua = ua || navigator.userAgent;
 
-            if (/Win(16|9[x58]|NT( [1234]| 5\.0| [^0-9]|[^ -]|$))/.test(ua) ||
-                    /Windows ([MC]E|9[x58]|3\.1|4\.10|NT( [1234]\D| 5\.0| [^0-9]|[^ ]|$))/.test(ua) ||
-                    /Windows_95/.test(ua)) {
-                /**
-                 * Officially unsupported platforms are Windows 95, 98, ME, NT 4.x, 2000
```

Comment:
These operating systems are now so old it's difficult to make a case for keeping this logic. I can't find any VMs services that still support these systems, which makes testing pretty much impossible. All this is doing imho is increasing the chances of false positives.

Reply:
I took a look in GA and less than 0.03% of Windows visitors over the past year have been on these systems combined. As such I think we're ok to drop this conditional.